lib/rules/multiline-comment-style.js

```diff
@@ -22,7 +22,21 @@ module.exports = {
         },
 
         fixable: "whitespace",
-        schema: [{ enum: ["starred-block", "separate-lines", "bare-block"] }],
+        schema: [
+            {
+                enum: ["starred-block", "separate-lines", "bare-block"]
+            },
+            {
+                type: "object",
+                properties: {
+                    allowJsDoc: {
+                        type: "boolean",
+                        default: true
+                    }
+                },
+                additionalProperties: false
+            }
+        ],
```

Comment:
Does the new option apply only when the string option is `"separate-lines"`? If so, to avoid confusion we should enforce in the schema that the option can be specified only with `"separate-lines"`.

Reply:
Yes it only applies to `separate-lines`. I've looked at existing rules and couldn't find an example of enforcing an option associated with another option. Do you have any example or reference I could look at to implement this?

Reply:
Ok now that I see your commit I get it. Thanks for that